docs/api/session.md

```diff
@@ -268,7 +268,7 @@ Returns:
 * `event` Event
 * `details` Object
   * `deviceList` [HIDDevice[]](structures/hid-device.md)
-  * `frame` [WebFrameMain](web-frame-main.md)
+  * `frame` [WebFrameMain](web-frame-main.md) | null
```

Comment:
Would it make sense to copy/paste the explanation for when the `frame` can be null for these APIs?

> May be `null` if accessed after the frame has either navigated or been destroyed.

Reply:
[c1486c5](https://github.com/electron/electron/pull/43473/commits/c1486c5181a2cbb1161f34d12693e6e711052c21)